scripts/gulp/tasks/browserAdapters.ts

```diff
@@ -20,7 +20,7 @@ export async function createChrome(): Promise<Browser> {
   const options = safeLaunchOptions();
   let browser: puppeteer.Browser;
   let attempt = 1;
-  while (!browser) {
+  while (!browser!) {
```

Comment:
only way how to get rid of TS error, "reffing variable before being set"

Reply:
why not just initialize the value with `undefined` ?